docs/index.rst

```diff
@@ -662,9 +687,6 @@ command scripts.
 +------------------------+-----------------------------+--------------------------------------------+---------------------------------------------------------+
 | %fileExtensionWithDot% |  --file-extension-with-dot= |  As above, without stripping the period.   |  .mov                                                   |
 +------------------------+-----------------------------+--------------------------------------------+---------------------------------------------------------+
-| %outputDirectory%      |  --output-directory=        |  The directory to which the output file    |  /path/to/access/copies                                 |
```

Comment:
This was accidentally documented twice - I removed the less useful of the two definitions.